package.json

```diff
@@ -246,6 +245,8 @@
     "test-integration-db2": "cross-env DIALECT=db2 yarn test-integration",
     "test-integration-ibmi": "cross-env DIALECT=ibmi yarn test-integration",
     "test-integration-snowflake": "cross-env DIALECT=snowflake yarn test-integration",
+    "test-integration-all": "yarn test-integration-mariadb && yarn test-integration-mysql && yarn test-integration-postgres && yarn test-integration-postgres-native && yarn test-integration-sqlite && yarn test-integration-mssql && yarn test-integration-db2 && yarn test-integration-ibmi && yarn test-integration-snowflake",
+    "----------------------------------------- all tests ---------------------------------------------": "",
```

Comment:
Do we want to add a `test-all-oldest` and `test-all-latest`?

Reply:
The databases use the same port, so the only difference is in which database version was started before running this command